redirects-list.json

```diff
@@ -19,7 +19,7 @@
   "^/((?:deb|rpm)/.+)                                                                     https://s3-us-east-2.amazonaws.com/dvc-s3-repo/$1 303",
   "^/(?:help|chat)/?$                                                                     https://discordapp.com/invite/dvwXA2N 303",
   "^/(?:docs|documentation)(/.*)?$                                                        /doc$1",
-  "^/doc/?$                                                                               /doc/home 307",
+  "^/doc/home/?$                                                                          /doc 307",
```

Comment:
Do we need this redirect, since the previous one was 307? Hopefully it's not cached in browsers/ Not sure

Reply:
Only if someone put a link somewhere in the wild. I would probably not worry too much and remove it.